src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -707,21 +726,16 @@ public static void hoverInAppServerCfgFile(RemoteRobot remoteRobot, String hover
                 ContainerFixture popup;
                 switch (popupType.name()) {
                     case "DOCUMENTATION":
-                        popup = projectFrame.getDocumentationHintEditorPane();
+                        projectFrame.getDocumentationHintEditorPane();
                         break;
                     case "DIAGNOSTIC":
-                        popup = projectFrame.getDiagnosticPane();
+                        projectFrame.getDiagnosticPane();
```

Comment:
not clear what the lines `projectFrame.getDiagnosticPane()` and `projectFrame.getDocumentationHintEditorPane()` are doing in this context

Reply:
they are simply attempting to open the documentation or diagnostic popups themselves - there have been instances where the popups fail to open at all and this will catch that and re-try to open them until one does open